src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxIPv4InterfaceProperties.cs

```diff
@@ -34,9 +34,24 @@ public LinuxIPv4InterfaceProperties(LinuxNetworkInterface linuxNetworkInterface)
 
         private bool GetIsForwardingEnabled()
         {
-            // /proc/sys/net/ipv4/conf/<name>/forwarding
-            string path = Path.Combine(NetworkFiles.Ipv4ConfigFolder, _linuxNetworkInterface.Name, NetworkFiles.ForwardingFileName);
-            return StringParsingHelpers.ParseRawIntFile(path) == 1;
+            try
+            {
+                // /proc/sys/net/ipv4/conf/<name>/forwarding
+                string path = Path.Combine(NetworkFiles.Ipv4ConfigFolder, _linuxNetworkInterface.Name, NetworkFiles.ForwardingFileName);
```

Comment:
nit: Path.Join instead of Path.Combine

Reply:
@JeremyKuhne I assume Path.Join is faster as it doesn't handle quirks?

Reply:
Path.Combine also ends calling [JoinInternal ](https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/IO/Path.cs#L508) but Path.Internal doesn't check if the paths are rooted. So it could make sense to use Path.Join since we know the only rooted path is the first one, so therefore at the end it will just end up combining them.